src/status_im/ui/screens/chat/toolbar_content.cljs

```diff
@@ -2,13 +2,13 @@
   (:require [status-im.i18n.i18n :as i18n]
```

Comment:
same here ,if there are no changes except renaming better to revert this file, so we don't have changes in the old files